exporter/sumologicexporter/exporter.go

```diff
@@ -456,3 +456,25 @@ func getSignalURL(oCfg *Config, endpointURL string, signal pipeline.Signal) (str
 
 	return url.String(), nil
 }
+
+func sanitizeURL(urlString string) string {
+	leftIndex := strings.Index(urlString, "otlp/")
+	rightIndex := strings.LastIndex(urlString, "/v1/")
+	if leftIndex == -1 || rightIndex == -1 {
+		return urlString
+	}
+	s1 := urlString[0 : leftIndex+5]
+	s2 := nchars('*', (rightIndex - leftIndex - 5))
```

Comment:
Should there be a check on the values of the indices here, to ensure that right is greater than left by at least the difference required?